jota/src/main/java/jota/utils/IotaAPIUtils.java

```diff
@@ -65,52 +67,41 @@ public static String newAddress(String seed, int security, int index, boolean ch
                 // Get the corresponding keyIndex of the address
                 int keyIndex = 0;
                 int keySecurity = 0;
+                String signatureSeed = oseed;
```

Comment:
This doesn't support multi-seed inputs even though I believe this was your original intention?